4. User API/userAPI/src/main/kotlin/utility/ResponseUtil.kt

```diff
@@ -0,0 +1,7 @@
+package utility
+
+object ResponseUtil {
```

Comment:
you can make 2 methods, one to denote success response and another to denote failure response.